response_operations_ui/controllers/iac_controller.py

```diff
@@ -32,16 +32,15 @@ def get_iac_details(iac):
     return response.json()
 
 
-def get_latest_active_iac_code(cases, collection_exercises):
+def get_latest_case(cases, collection_exercises):
```

Comment:
Yeah can we add some inline comments or a docstring for clarity here please

Reply:
This is probably in the wrong file too, will try and make it a bit clearer